src/components/megaboard.js

```diff
@@ -7,7 +7,7 @@ class Megaboard extends React.Component {
   };
 
   isValidBoard(b) {
-    return this.boardStatus[b];
+    return this.state.boardStatus[b];
```

Comment:
THis is great!
![bitmoji](https://render.bitstrips.com/v2/cpanel/9666023-134307489_16-s1-v1.png?transparent=1&palette=1&width=246)